brian2/synapses/cythonspikequeue.pyx

```diff
@@ -5,10 +5,10 @@
 from libcpp.vector cimport vector
 from libcpp.pair cimport pair
 from libcpp.string cimport string
-
 import cython
 from cython.operator import dereference
 from cython.operator cimport dereference
+from cpython.pycapsule cimport PyCapsule_New, PyCapsule_GetPointer
```

Comment:
Yet another nitpick: I don't think we use `PyCapsule_GetPointer` anywhere :blush: